opm/parser/eclipse/EclipseState/Schedule/MSW/SegmentSet.cpp

```diff
@@ -21,6 +21,11 @@
 #include <cmath>
 #include <map>
 
+#ifdef _WIN32
+#define _USE_MATH_DEFINES 
+#include <math.h>
```

Comment:
From the suggested solutions, here is my prioritized list

1. define pi as constxpr
2. move _USE_MATH_DEFINES above #include <cmath>
3. build parameter